python/ray/tune/integration/docker.py

```diff
@@ -60,15 +60,24 @@ def _remote_path(self) -> Tuple[str, str]:
 
 class DockerSyncClient(SyncClient):
     """DockerSyncClient to be used by DockerSyncer.
+
     This client takes care of executing the synchronization
     commands for Docker nodes. In its ``sync_down`` and
     ``sync_up`` commands, it expects tuples for the source
     and target, respectively, for compatibility with docker.
+
+    Args:
+        should_bootstrap: Whether to bootstrap the autoscaler
+            cofiguration. This may be useful when you are
+            running into authentication problems; i.e.:
+            https://github.com/ray-project/ray/issues/17756.
+
     """
 
-    def __init__(self):
+    def __init__(self, should_bootstrap: bool = True):
         self._command_runners = {}
         self._cluster_config = None
+        self._should_bootstrap = should_bootstrap
```

Comment:
Won't it be easier to pass input through the cluster configuration file and set this during config?